tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,229 @@
+                  onClick={() => act("map")} >
+                  Map
+                </Button>
+                {!!upper_admin && (
+                  <Button.Checkbox
+                    ml={1}
+                    color="red"
+                    checked={!avmap}
+                    onClick={() => act("toggle_map")} >
+                    Disable{!avmap ? "d" : ""}
+                  </Button.Checkbox>
+                )}
+              </Box>
+              <Box mb={1}>
+                <Button
+                  disabled={!upper_admin || !avr}
+                  onClick={() => act("restart")} >
+                  Restart
+                </Button>
+                {!!upper_admin && (
+                  <Button.Checkbox
+                    ml={1}
+                    color="red"
+                    checked={!avr}
+                    onClick={() => act("toggle_restart")} >
+                    Disable{!avr ? "d" : ""}
+                  </Button.Checkbox>
+                )}
+              </Box>
+              <Box mb={1}>
+                <Button
+                  disabled={!upper_admin || !avm}
+                  onClick={() => act("gamemode")} >
+                  Gamemode
+                </Button>
+                {!!upper_admin && (
+                  <Button.Checkbox
+                    ml={1}
+                    color="red"
+                    checked={!avm}
+                    onClick={() => act("toggle_gamemode")} >
+                    Disable{!avm ? "d" : ""}
+                  </Button.Checkbox>
+                )}
+              </Box>
+            </Flex.Item>
+            <Flex.Item>
+              <Button disabled={!upper_admin} onClick={() => act("custom")}>
+                Create Custom Vote
+              </Button>
+            </Flex.Item>
+          </Flex>
+        </Collapsible>
+        <Collapsible title="View Voters">
+          <Box mt={2} width="100%" height={6} overflowY="scroll">
+            {voting.map(voter => {
+              return <Box key={voter}>{voter}</Box>;
+            })}
+          </Box>
+        </Collapsible>
+      </Section>
+    </Flex.Item>
+  );
+};
+
+// Display choices as buttons
+const ChoicesPanel = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, choices } = data;
+
+  let content;
+  if (choices.length === 0) {
+    content = "No choices available!";
+  }
+  // Single box for most normal vote types
+  else if ((choices.length < 10) | (mode === "custom")) {
+    content = (
+      <DisplayChoices
+        choices={choices}
+        tally="Votes:"
+        startIndex={0}
+        margin={1} />
+    );
+  } else {
+    // If there's both too much content, most likely gamemode
+    content = (
+      <Flex justify="space-between" direction="row">
+        <Flex direction="column">
+          <DisplayChoices
+            choices={choices.filter(
+              (choice, index) => index < choices.length / 2
+            )}
+            tally="|"
```

Comment:
I'm pretty sure you could up your game by making these Divider components. They look pretty fresh trust me

Reply:
Do you mean the tally using a divider or just using a solid divider? I will probably go through this tonight to see what you mean here

Reply:
I mean the tally prop here is equal to a divider instead of "|" so it uses the component. Think about how for sections the buttons prop holds button components that are slotted into the section